src/Microsoft.Identity.Web/TokenAcquisition.cs

```diff
@@ -421,6 +417,8 @@ public async Task ReplyForbiddenWithWwwAuthenticateHeaderAsync(IEnumerable<strin
             httpResponse.StatusCode = (int)HttpStatusCode.Forbidden;
 
             headers[HeaderNames.WWWAuthenticate] = new StringValues($"{Constants.Bearer} {parameterString}");
+
+            await Task.CompletedTask.ConfigureAwait(false); // we don't want to take a breaking change right now. will be for 2.0
```

Comment:
This is one of those cases where the task should be returned instead of awaited.

Reply:
yes...i talked w/ @jmprieur about this one and we are going to wait until we move to 2.* and take a breaking change. so this will come soon.